web/pgadmin/evaluate_config.py

```diff
@@ -47,7 +58,10 @@ def validate_config_variable(key, value):
 
 # Load distribution-specific config overrides
 try:
-    import config_distro
+    if 'CONFIG_DISTRO_FILE_PATH' in os.environ:
```

Comment:
What if 'CONFIG_DISTRO_FILE_PATH' not in the env? Does this logic work with the current implementation?

Reply:
This works with existing implementation. This is required only for building the docker image. But let me add else block too.

Reply:
Yeah, else block is required. Otherwise, it will not work where we have a distro file other than Docker env.